Sources/Rosetta/PlayMode/CardSets/BlackTempleCardsGen.cpp

```diff
@@ -568,6 +571,13 @@ void BlackTempleCardsGen::AddMage(std::map<std::string, CardDef>& cards)
     // GameTag:
     //  - FREEZE = 1
     // --------------------------------------------------------
+    power.ClearData();
+    power.AddPowerTask(std::make_shared<SetGameTagTask>(
+        EntityType::ENEMY_MINIONS, GameTag::FROZEN, 1));
```

Comment:
The logic of this card is wrong. You should read the text carefully. `Freeze "an enemy".`